sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/SqlAnalyzer.java

```diff
@@ -144,14 +144,15 @@ static AnalyzerOptions baseAnalyzerOptions() {
         .setEnabledLanguageFeatures(
             new HashSet<>(
                 Arrays.asList(
-                    LanguageFeature.FEATURE_NUMERIC_TYPE,
+                    LanguageFeature.FEATURE_CREATE_AGGREGATE_FUNCTION,
```

Comment:
FYI This list has an internal equivalent for the Dataflow SQL validator. Please update it to match.

Reply:
Done.